gapir/cc/context.cpp

```diff
@@ -505,6 +505,22 @@ void Context::registerCallbacks(Interpreter* interpreter) {
         }
       });
 
+  interpreter->registerBuiltin(
+      Vulkan::INDEX, Builtins::ReplayDestroyVkInstance,
+      [this](uint32_t label, Stack* stack, bool) {
+        GAPID_DEBUG("[%u]replayDestroyVkInstance()", label);
+        if (mVulkanRenderer != nullptr) {
+          auto* api = mVulkanRenderer->getApi<Vulkan>();
+          return api->replayDestroyVkInstance(stack);
+        } else {
```

Comment:
The `else` may be omitted

Reply:
It can, but I wanted to keep it consistent with the rest.